models/linkages/R/read.restart.LINKAGES.R

```diff
@@ -0,0 +1,26 @@
+read.restart.LINKAGES <- function(outdir,run.id,time,IC,prior,spinup){
+  
+  forecast = IC
+  nens = nrow(IC)
+  ens = list()
+  
+  for(i in 1:nens){
```

Comment:
I don't think we really talked about it. I did that because I figured you always want to read all of the ensemble members, but I can also see why you would want it to be the other way. This change would also fix the dimensionality issue so we wouldn't have to pass IC correct?